src/mongoose-cleaner.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
```

Comment:
We might want to install and import only `lodash.omit` and `lodash.mapValues` to greatly reduce this module's size?